samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -8,17 +8,19 @@ nRF9160: HTTP application update
    :depth: 2
 
 The HTTP application update sample demonstrates how to do a basic firmware over-the-air (FOTA) update.
-It uses the :ref:`lib_fota_download` library to download a file from a remote server and write it to flash.
+It uses the :ref:`lib_fota_download` library to download two image files from a remote server and program them to flash memory.
```

Comment:
@jonathannilsen Please check if this line is actually correct

Reply:
Looks correct